mkdocs/docs/experiments/categories/pods/common-tunables-for-pod-experiments.md

```diff
@@ -200,6 +200,6 @@ spec:
     spec:
       components:
         env:
-        - name: NODE_LABEL
```

Comment:
Why are we removing the `NODE_LABEL` env?

Reply:
This was a mistake in this [PR](https://github.com/litmuschaos/litmus/pull/3553/files), the file name provided was incorrect, so running embedmd made the changes. Fixed the filename now.